docs/core/setup/linux-package-manager-opensuse423.md

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
This distro should be removed now as it has reached end of support. We just removed it from the website.